styles/screen.css

```diff
@@ -0,0 +1,27 @@
+.screen {
+  width: 388px;
```

Comment:
please avoid using exact units for width and height in pixels
rem is better, like you have font-size in keyboard class. coz our UI carries pretty much text information.